### PR TITLE
deps: update release-please to 14.1.0

### LINKED ISSUE #4157: Release please not proposing a release PR for google-cloud-java monorepo
Trying to implement release-please for [google-cloud-java ](https://github.com/googleapis/google-cloud-java) but release please is not proposing a PR after "releasable units" are committed to the default branch. CLI states "timeout".

### PR DESCRIPTION
**Pull request by chingor13**
Fixes #4157

### CODE REVIEW

**Comment by chingor13** [comment]
We need bot-config-utils to also handle the `format` fields from the JSON schema spec

File: packages/policy/package.json [hunk 1]
```diff
@@ -51,7 +51,9 @@
     "smee-client": "^1.2.2",
```
Comment (tmatsuo) [inline packages/policy/package.json]: unrelated file?

File: packages/policy/package-lock.json [hunk 2]
```diff
@@ -1,8349 +1,8 @@
 {
```
Comment (tmatsuo) [inline packages/policy/package-lock.json]: unrelated file?

**Review by tmatsuo — APPROVED** [review]

File: packages/policy/package.json (hunk 1 above)
Comment (chingor13) [reply packages/policy/package.json]: these are already removed
